src/background/paused.js

```diff
@@ -11,41 +11,42 @@
 
 import { store } from 'hybrids';
 
-import Options, { GLOBAL_PAUSE_ID } from '/store/options.js';
+import Options, {
+  GLOBAL_PAUSE_ID,
+  FILTERING_MODE_GHOSTERY,
+} from '/store/options.js';
 import * as OptionsObserver from '/utils/options-observer.js';
 import ManagedConfig, { TRUSTED_DOMAINS_NONE_ID } from '/store/managed-config';
 
 import {
   getDynamicRulesIds,
   PAUSED_ID_RANGE,
   PAUSED_RULE_PRIORITY,
+  ALL_RESOURCE_TYPES,
 } from '/utils/dnr.js';
 
 // Pause / unpause hostnames
 const PAUSED_ALARM_PREFIX = 'options:revoke';
 
-const ALL_RESOURCE_TYPES = [
-  'main_frame',
-  'sub_frame',
-  'stylesheet',
-  'script',
-  'image',
-  'font',
-  'object',
-  'xmlhttprequest',
-  'ping',
-  'media',
-  'websocket',
-  'webtransport',
-  'webbundle',
-  'other',
-];
+OptionsObserver.addListener(async function pausedSites(options, lastOptions) {
+  if (options.filteringMode !== FILTERING_MODE_GHOSTERY) {
```

Comment:
`options.filteringMode === FILTERING_MODE_ZAP` - should be safer in case we introduce more modes